content/webapp/utils/iiif/v3/index.ts

```diff
@@ -58,6 +58,7 @@ export function getMultiVolumeLabel(
   return stringAtIndex1 === itemTitle ? stringAtIndex0 : stringAtIndex1;
 }
 
+// TODO: Should we make this be the same as getAudioVideoLabel but for Video as well etc?
```

Comment:
@gestchild can you help me out on how this is used where we want to return the label as `undefined` if it's `"-"` instead of using a title override?

Reply:
I'm not sure now. I thought we returned undefined for the thumbnails of books in the grid view, but it looks to be displaying the "-": https://wellcomecollection.org/works/jm4kaz67/items

Reply:
That seems to come from here and is over 3 years old: https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/components/IIIFViewer/IIIFCanvasThumbnail.tsx#L174

I won't play around with it as it looks like it also needs the label to be of format `InternationalString`, so will remove the TODO and keep my helper function - something to consider sorting maybe one day!